webapp/app.py

```diff
@@ -63,10 +63,57 @@ def results():
     return flask.render_template("careers/results.html", **context)
 
 
-@app.route("/careers/<department>", methods=["GET", "POST"])
+@app.route("/careers/thank-you", methods=["POST"])
+def careers_thank_you():
+    messages = []
+    job_id_list = flask.request.form.get("applicationJobIdList").split(",")
+    job_title_list = flask.request.form.get("applicationJobTitleList").split(
+        ","
+    )
+    print(flask.request.form)
+    i = 0
+    while i < len(job_id_list):
+        response = submit_to_greenhouse(
+            flask.request.form, flask.request.files, job_id_list[i]
+        )
+        if response.status_code == 200:
+            messages.append(
+                {
+                    "type": "success",
+                    "title": f"{job_title_list[i]}",
+                    "text": ("Successfully submitted."),
+                }
+            )
+        else:
+            messages.append(
+                {
+                    "type": "error",
+                    "title": f"{job_title_list[i]}",
+                    "text": (
+                        f"Error {response.status_code}. {response.reason}."
+                        "<br /><a href='/careers/{job_id_list[i]}' "
+                        "style='padding-left: 2rem;'>"
+                        "Please try to apply to this job again!</a>"
+                    ),
+                    "response": response,
+                }
+            )
+        i = i + 1
+        # time.sleep(60)
+    if len(messages) > 0:
+        return flask.render_template(
+            "careers/thank-you.html", messages=messages
+        )
+    else:
+        return flask.redirect("/careers")
+
+
+@app.route(
+    "/careers/<regex('[a-z-]*[a-z][a-z-]*'):department>",
```

Comment:
I don't think this is what you are searching for.

What is the format of the departemnts?